dtests/src/test/java/io/snappydata/hydra/complexdatatypes/complexDataTypes.bt

```diff
@@ -1,54 +1,126 @@
   B=lead leadHosts=2 leadVMsPerHost=1 leadThreadsPerVM=2
   C=locator locatorHosts=3 locatorVMsPerHost=1 locatorThreadsPerVM=1
   D=worker workerHosts=3 workerVMsPerHost=1 workerThreadsPerVM=1
+  dataFilesLocation="/export/shared/QA_DATA/Complextypes_Data/ArrayType/"
   numRowsValidation = true
   fullResultSetValidation = true
   isSmokeRun = false
 
-io/snappydata/hydra/complexdatatypes/structTypes.conf
+
+  io/snappydata/hydra/complexdatatypes/mapTypes.conf
   A=snappyStore snappyStoreHosts=3 snappyStoreVMsPerHost=1 snappyStoreThreadsPerVM=1
   B=lead leadHosts=2 leadVMsPerHost=1 leadThreadsPerVM=2
   C=locator locatorHosts=3 locatorVMsPerHost=1 locatorThreadsPerVM=1
   D=worker workerHosts=3 workerVMsPerHost=1 workerThreadsPerVM=1
+  dataFilesLocation="/export/shared/QA_DATA/Complextypes_Data/MapType/"
   numRowsValidation = true
   fullResultSetValidation = true
   isSmokeRun = false
 
-io/snappydata/hydra/complexdatatypes/arrayOfStructTypes.conf
+
+   io/snappydata/hydra/complexdatatypes/smartConnectorMapTypes.conf
+   A=snappyStore snappyStoreHosts=3 snappyStoreVMsPerHost=1 snappyStoreThreadsPerVM=1
+   B=lead leadHosts=2 leadVMsPerHost=1 leadThreadsPerVM=2
+   C=locator locatorHosts=3 locatorVMsPerHost=1 locatorThreadsPerVM=1
+   D=worker workerHosts=3 workerVMsPerHost=1 workerThreadsPerVM=1
+   dataFilesLocation="/export/shared/QA_DATA/Complextypes_Data/MapType/"
+   numRowsValidation = true
+   fullResultSetValidation = true
+   isSmokeRun = false
+*/
+
+  io/snappydata/hydra/complexdatatypes/structTypes.conf
   A=snappyStore snappyStoreHosts=3 snappyStoreVMsPerHost=1 snappyStoreThreadsPerVM=1
   B=lead leadHosts=2 leadVMsPerHost=1 leadThreadsPerVM=2
   C=locator locatorHosts=3 locatorVMsPerHost=1 locatorThreadsPerVM=1
   D=worker workerHosts=3 workerVMsPerHost=1 workerThreadsPerVM=1
+  dataFilesLocation="/export/shared/QA_DATA/Complextypes_Data/StructType/"
   numRowsValidation = true
   fullResultSetValidation = true
   isSmokeRun = false
 
-io/snappydata/hydra/complexdatatypes/allMixedTypes.conf
-    A=snappyStore snappyStoreHosts=3 snappyStoreVMsPerHost=1 snappyStoreThreadsPerVM=1
-    B=lead leadHosts=2 leadVMsPerHost=1 leadThreadsPerVM=2
-    C=locator locatorHosts=3 locatorVMsPerHost=1 locatorThreadsPerVM=1
-    D=worker workerHosts=3 workerVMsPerHost=1 workerThreadsPerVM=1
-    numRowsValidation = true
-    fullResultSetValidation = true
-    isSmokeRun = false
+
+  io/snappydata/hydra/complexdatatypes/smartConnectorStructTypes.conf
+  A=snappyStore snappyStoreHosts=3 snappyStoreVMsPerHost=1 snappyStoreThreadsPerVM=1
+  B=lead leadHosts=2 leadVMsPerHost=1 leadThreadsPerVM=2
+  C=locator locatorHosts=3 locatorVMsPerHost=1 locatorThreadsPerVM=1
+  D=worker workerHosts=3 workerVMsPerHost=1 workerThreadsPerVM=1
+  dataFilesLocation="/export/shared/QA_DATA/Complextypes_Data/StructType/"
+  numRowsValidation = true
+  fullResultSetValidation = true
+  isSmokeRun = false
+
+/*
```

Comment:
Please resolve the commented code.

Reply:
Remove the comments. All the tests needs to be in master.